packages/pagination/src/mongo-pagination-param.decorator.ts

```diff
@@ -8,7 +8,10 @@ const DEFAULT_NUMBER_OF_RESULTS: number = 10;
  * Mongo query
  */
 export interface MongoPagination {
-  filter: {};
+  filter: {
+    // tslint:disable-next-line: no-any
+    [key: string]: any;
```

Comment:
You could use `unknown` instead of `any`

Reply:
I disagree, `unknown` is used to say to the developer:

> It could be any values, but you must check the type before asserting

In that case, this is a Mongo query filter, we don't need this check in my opinion

Reply:
Ok anyway the filter is coming from a query, so the type check won't be useful